microk8s-installation.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+sudo snap install microk8s --classic
+sudo microk8s status --wait-ready
+sudo microk8s enable dns hostpath-storage registry
+sudo usermod -a -G microk8s $USER
+sudo chown -f -R $USER ~/.kube
+sudo microk8s kubectl -n kube-system rollout status -w deployment/hostpath-provisioner
+sudo microk8s kubectl -n kube-system rollout status -w deployment/coredns
+sudo microk8s kubectl -n container-registry rollout status -w deployment/registry
+sg microk8s -c "juju bootstrap microk8s k8s-ctrl"
+juju add-model testing
+docker load --input gunicorn/gunicorn.tar
```

Comment:
Where does the tar come from ?

Reply:
From the build image workflow, it builds the image in that location.

Reply:
the thing is that, as I'm using lxd as a provider, I have to load the image manually

Reply:
Wanted to make sure I got it properly, make sense